examples/e-commerce/src/app/clear-filters-mobile/clear-filters-mobile.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Inject, forwardRef } from '@angular/core';
+import { BaseWidget, NgAisInstantSearch } from 'angular-instantsearch';
+import { connectClearRefinements } from 'instantsearch.js/es/connectors';
+
+@Component({
+  selector: 'app-clear-filters-mobile',
+  template: `
+    <div class="ais-ClearRefinements">
+      <button
+        type="button"
+        class="ais-ClearRefinements-button"
+        (click)="handleClick($event)"
+      >
+        Reset filters
```

Comment:
This element shouldn't have the image prefixed in the CSS.

Reply:
5cca160